subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -15,11 +15,11 @@
 #include "ll_sw/pdu.h"
 #include "radio_nrf5.h"
 
-#include "nrf_radio.h"
-#include "nrf_rtc.h"
-#include "nrf_ccm.h"
-#include "nrf_timer.h"
-#include "nrf_ppi.h"
+#include <nrfx/hal/nrf_radio.h>
```

Comment:
why are you adding the full path? was there any conflict with the other file names? (this will break in nrf52_bsim)

Reply:
For consistency, mainly. We use "hal/" in new or revised code to differentiate between nrfx HAL and drivers (_<nrf_timer.h>_ and _<nrfx_timer.h>_ look too similar). Right now "ext/hal/nordic/nrfx/hal" is also added to the include paths to keep legacy code compiling but it is planned to be removed at some point.
I added additionally the "nrfx/" part here to emphasize that it is the HAL from nrfx, to avoid confusion with the HAL defined for the controller itself. Would it work for you without this part?

Reply:
I would need to replicate your directories, now the nrfx hal code is just the directly in: https://github.com/BabbleSim/ext_NRF52_hw_models/tree/master/src/nrfx_hal

Reply:
I see. Then removing "nrfx/" won't change much. You'd need to reorganize the directories anyway.
But would the change from "nrfx_hal" to "nrfx/hal" make much trouble to you?
I'd prefer to stay with the path in the current form. What is the opinion of the others?

Reply:
> "nrfx/hal" make much trouble to you? 

For me it is just some time to do it, and so it will be for everybody who is using this nrf52_bsim board right now (as it is unfortunately not merged yet). They will need to both merge a new version of the board and pick a new version of the HW models by hand.

Reply:
One comment here: The `nrfx/` folder is not part of your repository structure, but part of Zephyr's folder structure. If/when that is moved to another place (say due to multirepo) the nRFx may not be placed in a `nrfx/` folder anymore. In general that include path is `<nrfx root path>/hal/nrf_radio.h`, or?

Reply:
Yes, it is `<nrfx root path>/hal/nrf_radio.h`. As I wrote before, I added `nrfx/` to avoid confusion with the HAL for the controller - a few lines above there is inclusion of `"hal/radio.h"`.

Reply:
So, should that include line then not have the `nrfx/` part?

Reply:
I'd prefer to keep this `nrfx/` part. To clearly indicate that it is a different HAL from the one for the controller (and I think it's rather unlikely that nrfx can end up in a different folder than `nrfx/`).
But I won't insist on it as this is more of aesthetic aspect.
@cvinayak @ioannisg @carlescufi Any opinions? I think we should aim for merging this soon.

Reply:
I'm happy to keep the `nrfx/` path to make it very clear. When we go to multirepo we can easily fix this if necessary.